ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -158,6 +163,10 @@ func (scbuilder *SiteConfigBuilder) getClusterCRs(clusterId int, siteConfigTemp
 						clusterCRs = append(clusterCRs, instantiatedCR)
 					}
 				} else {
+					if kind == "BareMetalHost" {
+						// Ironic inspection is enabled by default, delete the inspect annotation if it is not disabled
```

Comment:
Is Ironic inspection disabled by default in all versions of ACM/BMO that we expect to run on (2.5, 2.6, 2.7+)? Or does the new "inspect" option only apply to ACM 2.6+ or 2.7+?

Reply:
The Ironic inspection is enabled by default in BMO, and it can be disabled by adding "inspect.metal3.io: disabled" annotation to BMH.  This was introduced for ZTP/AI deployment (i.e. ACM2.5, ACM2.6). In ACM 2.7 + and ocp version 4.11+, the converged flow will be the default behavior and the disable annotation should not be present.